cluster/kube-down.sh

```diff
@@ -10,11 +10,11 @@
 #
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either expressed or implied.
```

Comment:
You can't change the license text. (Also the original text is correct even if it wasn't legal text).